packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -179,6 +197,16 @@ export const EditTopicModal = ({
               label="Delete topic"
             />
           </div>
+          {image && (
+            <div>
+              <CWButton
+                buttonType="destructive"
+                buttonHeight="sm"
+                label="Remove image"
+                onClick={() => onDeleteImageClick()}
```

Comment:
I dont think this button is necessary if we have editor in place. Especially that it still doesn't work for more than one image.

Reply:
@masvelio Weird, I didn't see the editor that way when I was working on it, but I'll look into it. 
I also thought about removing the "Remove image" button, but then thought that it was an easy way for someone to remove an image if they didn't know what an image in markdown looked like and didn't know exactly what information/text to specifically delete in the editor.

Reply:
@masvelio Lol, true. Okay I'll remove it.